src/main/java/com/example/bookshop/model/User.java

```diff
@@ -5,19 +5,29 @@
 import jakarta.persistence.GeneratedValue;
 import jakarta.persistence.GenerationType;
 import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.JoinTable;
+import jakarta.persistence.ManyToMany;
 import jakarta.persistence.Table;
 import jakarta.validation.constraints.NotNull;
-import lombok.Data;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.Set;
+import lombok.Getter;
 import lombok.NonNull;
+import lombok.Setter;
 import org.hibernate.annotations.SQLDelete;
 import org.hibernate.annotations.Where;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
 
 @Entity
 @Table(name = "users")
 @SQLDelete(sql = "UPDATE users SET is_deleted = true WHERE id = ?")
-@Where(clause = "is_deleted = false")
-@Data
-public class User {
+@Where(clause = "status <> 'DELETED'")
```

Comment:
```suggestion
@SQLRestriction(value = "status <> 'DELETED'")
```

`@Where` is deprecated

Reply:
I can't find @SQLRestriction annotation.  I tried update hibernate but stil IntlijIdea can't find this annotation